RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingCustomToast/mobileAppointmentBookingCustomToast.html

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div class="slds-notify_container custom-toast-container">
+        <div class={toastClass} id="variant-container" role="status">
+        <span class="slds-assistive-text">{variant}</span>
+            <button class="slds-button slds-button_icon slds-button_icon-inverse" title="Close" onclick={handleCloseToast}>
+                <svg class="slds-button__icon slds-button__icon_large" aria-hidden="true">
+                <use xlink:href="/_slds/icons/utility-sprite/svg/symbols.svg#close"></use>
+                </svg>
+                <span class="slds-assistive-text">Close</span>
+            </button>
+        <div class="slds-notify__content content-container">
```

Comment:
This line generates the following SLDS error:

> Deprecated, or invalid, utility class: "slds-notify__content" SLDS(0)

Reply:
@khawkins it's is still supported and in 244 the toast component will be removed entirely for a different design so I'm keeping this for now

https://www.lightningdesignsystem.com/components/toast/